systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -240,4 +240,17 @@ public static boolean checkLogForJSONFormat(Map<String, String> pods, String con
         }
         return isJSON;
     }
-}
\ No newline at end of file
+
+    /**
+     /**
+     * Method for check if test is allowed on current Kubernetes version
+     * @param desiredKubernetesVersion kubernetes version which test needs
+     * @return true if test is allowed, false if not
+     */
+    public static boolean isAllowedOnCurrentK8sVersion(String desiredKubernetesVersion) {
```

Comment:
I would probably call the paramater again `maxKubernetesVersion` or something like that to make it more clear what it does and how it works.